news/31.feature

```diff
@@ -0,0 +1,3 @@
+- Upgrade and pin Vale to v3. @stevepiercy
+- Follow some of Vale's suggestions and fix errors and warnings. @stevepiercy
+- Add GitHub Workflow to build docs and run checks on links, spelling, grammar, and style. @stevepiercy
```

Comment:
Correct me if I'm wrong, but I think towncrier adds a bullet point for each file. You have to create multiple files if you want multiple bullet points in the generated changelog.

Reply:
Actually I think I can fake it by removing the first bullet point. Let's experiment!